src/main/java/org/motechproject/mots/domain/MessageLog.java

```diff
@@ -6,6 +6,10 @@
 import javax.persistence.Table;
 import lombok.NoArgsConstructor;
 
+/**
+ *  This class is used to log {@link Message} event to the db. It is a concrete implementation of
```

Comment:
I don't think event is the right word, it's all used to track the progress of the course/module.